### ISSUE TITLE
Performance warning when using Table.drop/pick_files inplace

### ISSUE THREAD

**Issue by hagenw** [audeering/audformat#202]
When using the `inplace=True` argument with `audformat.Table.pick_files()` or `audformat.Table.drop_files()` you get the following warning:

```
.../python3.8/site-packages/audformat/core/table.py:374: PerformanceWarning: dropping on a non-lexsorted multi-index without a level parameter may impact performance.
  self.df.drop(index, inplace=True)
```

Sounds we can fix this by using a level parameter.

**Comment by hagenw** [comment]
It's not due to `inplace=True`, the same warning appears when using `inplace=False`.

**Comment by hagenw** [comment]
This should have been fixed in https://github.com/audeering/audformat/pull/175 already. It can be checked by updating to `audformat` 0.14.3 which includes the changes.

**Comment by hagenw** [comment]
I would like to test it first, but I have access to the compute servers only in around 1 hour ;)

**Comment by hagenw** [comment]
It is indeed fixed in `audformat` 0.14.3
